tests/compat.sh

```diff
@@ -236,6 +236,14 @@ reset_ciphersuites()
     G_CIPHERS=""
 }
 
+check_translation()
+{
+    if [ $? -eq 1 ]; then
```

Comment:
Since you're passing `$?` and `$T` as positional parameters, you might as well refer to them as `$1` and `$2` inside the function. (Alternatively, call the function with no parameters, and keep accessing the variables with their global name, but I think it's less clean: for `$?` there's a risk of over-writing it unintentionally, and for `$T` that creates coupling in that everyone must use the same name.)

Also, I think the test should be `-ne 0`. We want to catch all errors, even unexpected ones. (For example, imagine the file is accidentally removed/renamed, `python3` will return 2 and we want to catch that too.)

Reply:
I tried using $2 originally but it takes each word of the error message e.g
```
"help I errored"
echo "$2"
> help
```
I thought maybe $T would be nicer than echoing every paremeter >2.
any other suggestion?